listMembersOfaJobGroups.groovy

```diff
@@ -0,0 +1,17 @@
+import nectar.plugins.rbac.groups.*
+
+def job = Jenkins.instance.items.each{it instanceof Job}.find{it.name == 'CopyG'}
```

Comment:
I think this could be reduced to:

def job = Jenkins.instance.getItems(Job.class).find{it.name == 'CopyG'}